ejemplo.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>The Cheese Guardian</title>
+	<link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+	<section>
```

Comment:
first of all you could put the div without wrapping it inside a section, this would make the HTML simpler, but this is something style related so if possible try to avoid adding stuff in the html for styling, there are moments like the ```vertical-separator``` you created bellow where you have no choice but here you can use a border-top in the section and this way you will have the top orange line.